tests/test_mig_shared_configuration.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# --- BEGIN_HEADER ---
+#
+# test_mig_install_geneateconfs - unit test of generateconfs
```

Comment:
nit-pick: apart from the typo, this line should match module name and description